libs/src/service-selection/ServiceSelection.ts

```diff
@@ -26,15 +26,15 @@ interface GetServices {
 }
 
 interface Decision {
-  stage: DECISION_TREE_STATE
+  stage: string
   val?: unknown
 }
 
 interface ServiceSelectionConfig {
   // services from this list should not be picked
-  blacklist?: Set<string>
+  blacklist?: Set<string> | undefined
```

Comment:
weirdly enough it doesn't. so ?: means that it may not be provided, but if you do `{blacklist: undefined}` then it complains... can double check that and send you a screenshot. if we feel like that case shouldn't, be handled, and we can just remove the explicit undefined part, happy to do so

Reply:
not sure what the desired behavior of this is tbh, but this is probably fine

Reply:
this is the issue im seeing: 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/8230000/161648052-f3fc5f0a-ceef-4b7e-8ef7-70bf2341e7c5.png">
in the `super` call in CreatorNodeSelection for ServiceSelection

Reply:
gotcha - makes sense how you have it